opensrp-reveal/src/test/java/org/smartregister/reveal/validators/WithinOperationAreaValidatorTest.java

```diff
@@ -0,0 +1,88 @@
+package org.smartregister.reveal.validators;
+
+import com.mapbox.geojson.Feature;
+import com.mapbox.mapboxsdk.camera.CameraPosition;
+import com.mapbox.mapboxsdk.geometry.LatLng;
+import com.mapbox.mapboxsdk.maps.MapboxMap;
+
+import org.junit.Before;
+import org.junit.Rule;
+import org.junit.Test;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnit;
+import org.mockito.junit.MockitoRule;
+import org.powermock.reflect.Whitebox;
+import org.smartregister.reveal.BaseUnitTest;
+import org.smartregister.reveal.util.TestingUtils;
+import org.smartregister.reveal.view.RevealMapView;
+
+import static org.junit.Assert.assertFalse;
+import static org.junit.Assert.assertNotNull;
+import static org.junit.Assert.assertTrue;
+import static org.mockito.Mockito.when;
+
+/**
+ * Created by Richard Kareko on 4/16/20.
+ */
+
+public class WithinOperationAreaValidatorTest extends BaseUnitTest {
+
+    @Rule
+    public MockitoRule rule = MockitoJUnit.rule();
+
+    @Mock
+    RevealMapView revealMapView;
+
+    @Mock
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use explicit scoping instead of the default package private level](https://app.codacy.com/manual/OpenSRP/opensrp-client-reveal/pullRequest?prid=5361929)